recorder.go

```diff
@@ -7,8 +7,8 @@ import (
 	"strconv"
 	"time"
 
-	trace "cloud.google.com/go/trace/apiv1"
 	"github.com/golang/protobuf/ptypes/timestamp"
+	gax "github.com/googleapis/gax-go"
```

Comment:
It's a dependency of the protobuf client we are using so, not sure we can do much there.